ACE/ace/CDR_Base.cpp

```diff
@@ -1395,8 +1395,8 @@ ACE_CDR::Fixed ACE_CDR::Fixed::div_helper2 (const Fixed &rhs, Fixed &r) const
         top.digit (i, 0);
       for (int i = dig; i < this->digits_; ++i)
         bot.digit (i, 0);
-      bot.digits_ = dig;
-      top.scale_ += dig;
+      bot.digits_ = static_cast<Octet> (dig);
+      top.scale_ += bot.digits_;
```

Comment:
Test necessary for the change to bot.digits_?

Reply:
It's not really changing - the value `dig` is an int that we know is constrained to small positive values.  This adds that cast to silence a warning.